xzes40/transformer/doc/src/daemon.md

```diff
@@ -0,0 +1,47 @@
+# Application Daemon
+
+In order to preserve state in the in-memory cache, XZES has a long-running daemon.
+
+Here is roughly how that daemon works and how it recieves jobs.
+
+```
+ +----------------------+
+ |                      |
+ |Main Executable       |
+ |                      |
+ +-----+-----------^----+
+       |           |
++-------------------------------------------------------+
+|      |           |                                    |
+|      |           |                                    |
+|  +---v-----------+----+      +--------------------+   |
+|  |                    +------>                    |   |
+|  |Daemon Process      |      |Transformer         |   |
+|  |                    <------+                    |   |
+|  +--------------------+      +----+--------^------+   |
+|                                   |        |          |
+|                                   |        |          |
+|  +--------------------------------v--------+------+   |
+|  |                                                |   |
+|  |Cache                                           |   |
+|  |                                                |   |
+|  +------------------------------------------------+   |
+|                                                       |
++-------------------------------------------------------+
+```
+
+## The Daemon process startup
+
+1. When the `daemon` binary is executed it sets up a long running process and goes into a state of cybernation.
```

Comment:
This is supposed to say "Hibernation"

That said, "cybernation" is a pretty cool word.